convert2rhel/unit_tests/checks_test.py

```diff
@@ -489,15 +479,60 @@ def test_check_uefi_efi_detected(self):
             in checks.logger.critical_msgs[0]
         )
         if checks.logger.debug_msgs:
-            self.assertFalse(
-                "Converting BIOS system" in checks.logger.debug_msgs[0]
-            )
+            self.assertFalse("Converting BIOS system" in checks.logger.debug_msgs[0])
 
     @unit_tests.mock(os.path, "exists", lambda x: not x == "/sys/firmware/efi")
     @unit_tests.mock(checks, "logger", GetLoggerMocked())
     def test_check_uefi_bios_detected(self):
         checks.check_uefi()
         self.assertFalse(checks.logger.critical_msgs)
+        self.assertTrue("Converting BIOS system" in checks.logger.debug_msgs[0])
+
+
+class TestReadOnlyMountsChecks(unittest.TestCase):
+    @unit_tests.mock(checks, "logger", GetLoggerMocked())
+    @unit_tests.mock(
+        checks,
+        "get_file_content",
+        GetFileContentMocked(
+            data=[
+                "sysfs /sys sysfs rw,seclabel,nosuid,nodev,noexec,relatime 0 0",
+                "mnt /mnt sysfs rw,seclabel,nosuid,nodev,noexec,relatime 0 0",
+                "cgroup /sys/fs/cgroup/cpuset cgroup rw,seclabel,nosuid,nodev,noexec,relatime,cpuset 0 0",
+            ]
+        ),
+    )
+    def test_mounted_are_readwrite(self):
+        checks.check_readonly_mounts()
+        self.assertEqual(len(checks.logger.critical_msgs), 0)
+        self.assertEqual(len(checks.logger.debug_msgs), 2)
+        self.assertTrue(
+            "/mnt mount point is not read-only." in checks.logger.debug_msgs
+        )
+        self.assertTrue(
+            "/sys mount point is not read-only." in checks.logger.debug_msgs
+        )
+
+    @unit_tests.mock(checks, "logger", GetLoggerMocked())
+    @unit_tests.mock(
+        checks,
+        "get_file_content",
+        GetFileContentMocked(
+            data=[
+                "sysfs /sys sysfs rw,seclabel,nosuid,nodev,noexec,relatime 0 0",
+                "mnt /mnt sysfs ro,seclabel,nosuid,nodev,noexec,relatime 0 0",
+                "cgroup /sys/fs/cgroup/cpuset cgroup rw,seclabel,nosuid,nodev,noexec,relatime,cpuset 0 0",
+            ]
+        ),
+    )
+    def test_mounted_are_readonly(self):
+        self.assertRaises(SystemExit, checks.check_readonly_mounts)
+        self.assertEqual(len(checks.logger.critical_msgs), 1)
+        self.assertTrue(
+            "Stopping conversion due to read-only mount to /mnt directory"
+            in checks.logger.critical_msgs[0]
+        )
         self.assertTrue(
-            "Converting BIOS system" in checks.logger.debug_msgs[0]
+            "Stopping conversion due to read-only mount to /sys directory"
```

Comment:
Nit: In pytest you would be able to do all these tests as one and parameterized it:)